pkg/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/k8ssandra/cleaner"
+)
+
+var (
+	podNameSpaceEnvVar = "POD_NAMESPACE"
+)
+
+func main() {
```

Comment:
Is this something that the developers will run to perform a local cleanup?

Reply:
This is ran as part of the ``helm uninstall`` 's pre-delete hook